website/static/js/accountSettings.js

```diff
@@ -207,7 +207,22 @@ var UserProfileViewModel = oop.extend(ChangeMessageMixin, {
                     if (emails[i].address() === email.address()) {
                         this.emailInput('');
                         var addrText = $osf.htmlEscape(email.address());
-                        $osf.growl('<em>' + addrText  + '</em> added to your account.','You will receive a confirmation email at <em>' + addrText  + '</em>. Please check your email and confirm.', 'success');
+                            bootbox.confirm({
```

Comment:
Currently, if a user adds an email but presses `Cancel`, the address is still added to their account and we send them an email. This method needs to be refactored if we want to allow a user to actually cancel after reading this but before the new address is added to their account. If not, you could use `bootbox.alert` here.